### PR TITLE
Update cluster url

### PR DESCRIPTION
**Pull request by mhmxs**
First try of goreleaser

### CODE REVIEW

File: .github/.goreleaser-release.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  draft: true
+
+  # If set to auto, will mark the release as not ready for production
+  # in case there is an indicator for this in the tag e.g. v1.0.0-rc1
+  # If set to true, will mark the release as not ready for production.
+  # Default is false.
+  prerelease: auto
+
+  extra_files:
+    - glob: ./specs/cluster.yaml
```
Comment (nolancon) [inline .github/.goreleaser-release.yaml:57]: should this be `storageos-cluster.yaml` ?

Comment (nolancon) [inline .github/.goreleaser-release.yaml:57]: should this be `storageos-cluster.yaml` ?

File: cmd/plugin/cli/version.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	"github.com/spf13/cobra"
+)
+
+var Version string
+
+func VersionCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "version",
+		Args:         cobra.MinimumNArgs(0),
+		Short:        "Show the StorageOS version information",
```
Comment (nolancon) [inline cmd/plugin/cli/version.go:15]: does this display StorageOS version or `kubectl-storageos` version?

Comment (mhmxs) [reply cmd/plugin/cli/version.go:15]: kubectl-storageos version

Comment (nolancon) [reply cmd/plugin/cli/version.go:15]: Should probably just read `kubectl storageos version` then IMO

Comment (mhmxs) [reply cmd/plugin/cli/version.go:15]: I don't understand, sorry

**Review by nolancon — APPROVED** [review]

**Pull request merged**